### PR TITLE
Rav/beam master/250 snapshot

### PR DESCRIPTION
**Pull request by ravwojdyla**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -20,7 +20,7 @@ import Keys._
 import sbtassembly.AssemblyPlugin.autoImport._
 import com.typesafe.sbt.SbtGit.GitKeys.gitRemoteRepo
 
-val beamVersion = "2.4.0"
+val beamVersion = "2.5.0-SNAPSHOT"
```
Comment (nevillelyh) [inline build.sbt]: CI script will update this when building beam-master branch. We shouldn't change this.

File: version.sbt [hunk 2]
```diff
@@ -1 +1 @@
-version in ThisBuild := "0.5.2-SNAPSHOT"
+version in ThisBuild := "0.5.3-SNAPSHOT"
```
Comment (nevillelyh) [inline version.sbt]: We should undo this change too?

**Comment by ravwojdyla** [comment]
closing this for now, will wait for #1110 .
